server/lib/src/idm/server.rs

```diff
@@ -677,6 +693,55 @@ pub trait IdmServerTransaction<'a> {
         })
     }
 
+    fn process_uuid_to_identity(
+        &mut self,
+        uuid: &Uuid,
+        ct: Duration,
+        source: Source,
+    ) -> Result<Identity, OperationError> {
+        let anon_entry = self
+            .get_qs_txn()
+            .internal_search_uuid(UUID_ANONYMOUS)
+            .map_err(|e| {
+                admin_error!("Failed to validate ldap session -> {:?}", e);
+                e
+            })?;
+
+        let entry = if *uuid == UUID_ANONYMOUS {
+            anon_entry.clone()
+        } else {
+            self.get_qs_txn().internal_search_uuid(*uuid).map_err(|e| {
+                admin_error!("Failed to start auth ldap -> {:?}", e);
+                e
+            })?
+        };
+
+        if Account::check_within_valid_time(
+            ct,
+            entry
+                .get_ava_single_datetime(Attribute::AccountValidFrom)
+                .as_ref(),
+            entry
+                .get_ava_single_datetime(Attribute::AccountExpire)
+                .as_ref(),
+        ) {
+            // Good to go
+            let limits = Limits::default();
```

Comment:
I think this might be a mistake from when I added account policy, but I wonder if we should update the Limits here based on account policy from https://github.com/kanidm/kanidm/blob/master/server/lib/src/idm/server.rs#L889 ? Because these limits have impacts on search requests (and we currently don't implement paged search)

Reply:
Yes, makes sense. I have updated search limits.